serverless/src/samlRefreshToken/handler.js

```diff
@@ -24,8 +26,8 @@ const findLaunchpadTokenInHeaders = (headers) => {
  * Handles refreshing user's token
  * @param {Object} event Details about the HTTP request that it received
  */
-
 const samlRefreshToken = async (event) => {
+  // TODO This needs to be reimplemented to use the JWT
```

Comment:
Is this out of scope of this ticket?

Reply:
We have a ticket for refreshing the token, it wasn't written when I left this note, I'll add the ticket number